django/contrib/sessions/checks.py

```diff
@@ -0,0 +1,26 @@
+from django.core import checks
+from django.core.checks import register, Tags
+from django.conf import settings
+
+
+@register(Tags.sessions)
+def check_session_settings(**kwargs):
```

Comment:
Don't really need this function I think. It's easy enough to decorator other functions if they're added (as in django/core/checks/security.py)